backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5679120_sys_gh14645_UseDefaultBillToLocationAsOrderDefaultLocation_sys_config.sql

```diff
@@ -0,0 +1,4 @@
+-- 2023-02-23T13:01:53.284Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541597,'S',TO_TIMESTAMP('2023-02-23 15:01:53','YYYY-MM-DD HH24:MI:SS'),100,'U','Y','de.metas.order.impl.UseDefaultBillToLocationAsOrderDefaultLocation',TO_TIMESTAMP('2023-02-23 15:01:53','YYYY-MM-DD HH24:MI:SS'),100,'N')
```

Comment:
please add a description to the asysconfig, such that the user knows
- what it is about
- how the system actually behaves for Y and for N

Reply:
please use a proper entitiy-type - either an order-specific one or D

Reply:
please set the ConfigurationLevel to "Org". then one doesn't need to update this single record, but can override it with different values for different orgs